bitrise.yml

```diff
@@ -1,114 +1,49 @@
-format_version: 6
+format_version: "11"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
-app:
-  envs:
-  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/bitrise-io/bitrise-step-android-build-for-ui-testing.git
-  - BITRISE_SAMPLE_APP: https://github.com/bitrise-io/android-multiple-test-results-sample.git
-  - BITRISE_SAMPLE_APP_BRANCH: maintenance
-
 workflows:
-  go-tests:
+  check:
+    steps:
+      - git::https://github.com/bitrise-steplib/steps-check.git: { }
+
+  e2e:
     steps:
-    - go-list:
-    - golint:
-    - errcheck:
-        # ubuntu lts stacks have an old go version which causes errcheck command failure
-        run_if: |-
-          {{ not (enveq "BITRISEIO_STACK_ID" "linux-docker-android-lts") }}
-    - go-test:
-    
+      - git::https://github.com/bitrise-steplib/steps-check.git:
+          inputs:
+            - workflow: e2e
+
   test:
```

Comment:
why do we keep both `test` and `test_xcode_8`?

Reply:
It is used both in the current CI, I didn't want to change it. We can remove them after we do the unified CI cleanup anyway